### ISSUE TITLE
Do we need `users.permissions`?

### ISSUE THREAD

**Issue by kenjis** [codeigniter4/shield#125]
https://github.com/codeigniter4/shield/blob/5d7ec1808fc1a7ca9161886135f1a0b17eedceba/src/Database/Migrations/2020-12-28-223112_create_auth_tables.php#L18

**Comment by lonnieezell** [comment]
Yes. That holds the user-specific permissions instead of permissions held at the group level.

**Comment by kenjis** [comment]
It seems we do not use the column.
There is the `auth_groups_users` table.

**Comment by MGatner** [comment]
I think @kenjis meant `auth_permissions_users` - we are storing them in a separate table so the field on `users` isn't needed:

https://github.com/codeigniter4/shield/blob/develop/src/Authorization/Traits/Authorizable.php#L312-L315

**Comment by kenjis** [comment]
Yes, `auth_permissions_users`.
Remove `users.permissions`?

**Comment by lonnieezell** [comment]
Ah, yes, that was an old way I was trying, thinking I was "optimizing", but ended up decided just to do it right lol. It can be removed.

**Comment by MGatner** [comment]
Remove!
